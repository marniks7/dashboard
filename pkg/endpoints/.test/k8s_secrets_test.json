{
    "apiVersion": "v1",
    "items": [
        {
            "apiVersion": "v1",
            "data": {
                "ca.crt": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGMyVnkKZG1WeUxXTmhRREUyTmpJek1qSTJPRFF3SGhjTk1qSXdPVEEwTWpBeE9EQTBXaGNOTXpJd09UQXhNakF4T0RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRjMlZ5ZG1WeUxXTmhRREUyTmpJek1qSTJPRFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRUFVLQnFwbGErZFY1b3czcFNCSDVBMnhDdEdMdnZCWllvM2pkeURKSmYKaXdpYWJPaEFKL1l5cS9PcWNBZ3F5UXdLYXJIcU5RNkFkUHpIRkl4T2dERXdvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVRXeWloOG1oNGRkTDE2T1FJVzUwCjRYK1RPSmd3Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUlnVDZqdGxuWU4zZXNqTVMzSlExWDlYb1pYLzRLa2pCL1YKb2p2QlFQZ0o4b29DSVFEaUlRVHN5Umh5SDJkeVBkK2xCajMweGk1eHBSaVRyanV2WGhuOXJBNFdSQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K",
                "namespace": "dGVrdG9uLW5hbWVzcGFjZQ==",
                "token": "ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklqaHpWbXRXZG5aaFVUSlJRbUZ6VmtWRlIzUnBaak50YTJvNU0wNWFPVkUzYkZWRFRuaE9aR2RJWmpRaWZRLmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUowWld0MGIyNHRibUZ0WlhOd1lXTmxJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5elpXTnlaWFF1Ym1GdFpTSTZJbVJsWm1GMWJIUXRkRzlyWlc0dE5uRTBNalFpTENKcmRXSmxjbTVsZEdWekxtbHZMM05sY25acFkyVmhZMk52ZFc1MEwzTmxjblpwWTJVdFlXTmpiM1Z1ZEM1dVlXMWxJam9pWkdWbVlYVnNkQ0lzSW10MVltVnlibVYwWlhNdWFXOHZjMlZ5ZG1salpXRmpZMjkxYm5RdmMyVnlkbWxqWlMxaFkyTnZkVzUwTG5WcFpDSTZJbVUzT0dZMk1qSTFMVGRtTkdVdE5Ea3dNeTFoWVRjNExUWTNNRGd5TURaa016TXdaU0lzSW5OMVlpSTZJbk41YzNSbGJUcHpaWEoyYVdObFlXTmpiM1Z1ZERwMFpXdDBiMjR0Ym1GdFpYTndZV05sT21SbFptRjFiSFFpZlEuTVNhd2JQUGRpaE5rRF81QmpmcjNqSmNaX01wZm50YWRwVGtvdzB1UVh3R0VMeWZHa2N4Y2s0MWVGNFVoWE9zYnZXaVhWYXBwaXRzb0xtY1hCcE9kVm9ndU8yUVdYdTAxTHFzdHk1a3RzdFg3N3RpeTJRRXpGMlRrcjBMbW4wWnRISGEtb1VzX1VZSElSanVVa09Fd1hQbkRmT1FiMjd1VjlHOUFnQkUwVVZZLUpTUDgyaDVjSFhZYmhtanJKendNczhvekp5ckZMaG9GXzJwNE5GeVlXbS10bVJnYlV6V1RDUllNUFcyczBHcjBWVzlpRTlKakR2MVNscXcwQ1JadkJwbWt2a1o4Z1VseExaN3lyWU5Mdkg5S2JQcjUtYVdEMUtPdks5OU80eUVLU3dXUGlKVHdyTU9zeTllTWg4YXU1S09oTW5CR19obXpRbE1pald2T0Rn"
            },
            "kind": "Secret",
            "metadata": {
                "annotations": {
                    "kubernetes.io/service-account.name": "default",
                    "kubernetes.io/service-account.uid": "e78f6225-7f4e-4903-aa78-6708206d330e"
                },
                "creationTimestamp": "2022-09-05T00:47:14Z",
                "name": "default-token-6q424",
                "namespace": "tekton-namespace",
                "resourceVersion": "27862",
                "uid": "98a43c2e-2ab9-4c6f-ac99-7c757fe2e511"
            },
            "type": "kubernetes.io/service-account-token"
        },
        {
            "apiVersion": "v1",
            "data": {
                "kubeconfig": "YXBpVmVyc2lvbjogdjEKY2x1c3RlcnM6Ci0gY2x1c3RlcjoKICAgIGNlcnRpZmljYXRlLWF1dGhvcml0eS1kYXRhOiBMUzB0TFMxQ1JVZEpUaUJEUlZKVVNVWkpRMEZVUlMwdExTMHRDazFKU1VKbFJFTkRRVkl5WjBGM1NVSkJaMGxDUVVSQlMwSm5aM0ZvYTJwUFVGRlJSRUZxUVdwTlUwVjNTSGRaUkZaUlVVUkVRbWh5VFROTmRHTXlWbmtLWkcxV2VVeFhUbWhSUkVVeVRsUnJNVTlFU1hwT1ZGRjNTR2hqVGsxcVNYZFBSRUV3VFVSTmQwNVVWVEJYYUdOT1RYcEpkMDlFUVhoTlJFMTNUbFJWTUFwWGFrRnFUVk5GZDBoM1dVUldVVkZFUkVKb2NrMHpUWFJqTWxaNVpHMVdlVXhYVG1oUlJFVXlUbFJyTVU5RVNYcE9WRkYzVjFSQlZFSm5ZM0ZvYTJwUENsQlJTVUpDWjJkeGFHdHFUMUJSVFVKQ2QwNURRVUZTVm5adE4yRkpWM04wYVZVdlJsYzNjMnMyV1VaMk1sRkJVMUpwUW1zNVJGTXJNM2xuVmpaNkwySUtRMkl6WVRKaFlVRXhTbTAxTW5BMlNHOHdOV2xIVmt0TFpIRmxPSHBOZFVNM1VUUlpUVWhxUVRSRVpEZHZNRWwzVVVSQlQwSm5UbFpJVVRoQ1FXWTRSUXBDUVUxRFFYRlJkMFIzV1VSV1VqQlVRVkZJTDBKQlZYZEJkMFZDTDNwQlpFSm5UbFpJVVRSRlJtZFJWVk4xWVhwTU9VOW1SRGxIYmpZMFdVbERhemxRQ2tORWRuSnVRelIzUTJkWlNVdHZXa2w2YWpCRlFYZEpSRk5SUVhkU1owbG9RVXg2VDJ4RVdGVXpVbWx0YUZwTFIxcHBOVkZpUWpremJ6ZHZkbXRCU25RS05rWmpTekZGWlZZeVNqbFNRV2xGUVROd1Jpc3djSFJWY1RrMVlXVlJkbEUzWlV4WVZtWXhObUY1UlVwTk4xTTFSek5IV1ZWTk1tNUhPRFE5Q2kwdExTMHRSVTVFSUVORlVsUkpSa2xEUVZSRkxTMHRMUzBLCiAgICBzZXJ2ZXI6IGh0dHBzOi8vMC4wLjAuMDo2NDQzCiAgbmFtZTogazNkLXJ1bm5lcgpjb250ZXh0czoKLSBjb250ZXh0OgogICAgY2x1c3RlcjogazNkLXJ1bm5lcgogICAgdXNlcjogYWRtaW5AazNkLXJ1bm5lcgogIG5hbWU6IGszZC1ydW5uZXIKY3VycmVudC1jb250ZXh0OiBrM2QtcnVubmVyCmtpbmQ6IENvbmZpZwpwcmVmZXJlbmNlczoge30KdXNlcnM6Ci0gbmFtZTogYWRtaW5AazNkLXJ1bm5lcgogIHVzZXI6CiAgICBjbGllbnQtY2VydGlmaWNhdGUtZGF0YTogTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVSnJha05EUVZSbFowRjNTVUpCWjBsSlJHNVlVVTUwVlhKTmFHdDNRMmRaU1V0dldrbDZhakJGUVhkSmQwbDZSV2hOUWpoSFFURlZSVUYzZDFrS1lYcE9la3hYVG5OaFYxWjFaRU14YWxsVlFYaE9hbFUxVGxSbmVVMTZWVEJOUWpSWVJGUkplVTFFWjNkT1JFRjZUVVJWTVU1R2IxaEVWRWw2VFVSbmR3cE9SRUY2VFVSVk1VNUdiM2ROUkVWWVRVSlZSMEV4VlVWRGFFMVBZek5zZW1SSFZuUlBiVEZvWXpOU2JHTnVUWGhHVkVGVVFtZE9Wa0pCVFZSRVNFNDFDbU16VW14aVZIQm9Xa2N4Y0dKcVFscE5RazFIUW5seFIxTk5ORGxCWjBWSFEwTnhSMU5OTkRsQmQwVklRVEJKUVVKUE1tSlNWemRhVTJSNFlTOTFVRzRLYkdaNVMySkJWSGgxYVdGWk1XVXJkREpqTTA0dlJ6ZE1PRzUxWWtkWWVtRkxlVElyZEVsSFYxZENSblpOTVcxU1FVNDJVazk0ZUVsV1VHVjVOVGd2WXdwM1QzUm1TMW95YWxORVFrZE5RVFJIUVRGVlpFUjNSVUl2ZDFGRlFYZEpSbTlFUVZSQ1owNVdTRk5WUlVSRVFVdENaMmR5UW1kRlJrSlJZMFJCYWtGbUNrSm5UbFpJVTAxRlIwUkJWMmRDVTBscllYVmhUMWxMWWpjNVJGQTBUbEpGYTBOYVRtVXpNbk5aYWtGTFFtZG5jV2hyYWs5UVVWRkVRV2RPU2tGRVFrY0tRV2xGUVdvMmJURkJiazR4ZEVvd2JEUlFUM1JwUkdFM1RXZEhjbEV6ZVVoelFpczNWbTg0Tm05b01XWnlXVzlEU1ZGRGJTOVFhM1ZSVERKcmNHWlllZ3BEUWxScFExWXJUSEEwVFhFd2RHbHZkelZCVW1wcVFrbHNkRWhCT0ZFOVBRb3RMUzB0TFVWT1JDQkRSVkpVU1VaSlEwRlVSUzB0TFMwdENpMHRMUzB0UWtWSFNVNGdRMFZTVkVsR1NVTkJWRVV0TFMwdExRcE5TVWxDWkhwRFEwRlNNbWRCZDBsQ1FXZEpRa0ZFUVV0Q1oyZHhhR3RxVDFCUlVVUkJha0ZxVFZORmQwaDNXVVJXVVZGRVJFSm9jazB6VFhSWk1uaHdDbHBYTlRCTVYwNW9VVVJGTWs1VWF6RlBSRWw2VGxSUmQwaG9ZMDVOYWtsM1QwUkJNRTFFVFhkT1ZGVXdWMmhqVGsxNlNYZFBSRUY0VFVSTmQwNVVWVEFLVjJwQmFrMVRSWGRJZDFsRVZsRlJSRVJDYUhKTk0wMTBXVEo0Y0ZwWE5UQk1WMDVvVVVSRk1rNVVhekZQUkVsNlRsUlJkMWRVUVZSQ1oyTnhhR3RxVHdwUVVVbENRbWRuY1docmFrOVFVVTFDUW5kT1EwRkJWRUZuWWxKUlpFdGhhbkZrUTA5UU5tcEtkV3RvWW14ck5sSkNkVXR0ZWtwbU1rdGhXSE5HVUdsMUNrOVliemQxYzBneGVWbEplbHB4YmpaclNTdHhZVEV2WnpWTVMxVXpja0pQVkRCWGNrNVJWVk41UjFOUGJ6QkpkMUZFUVU5Q1owNVdTRkU0UWtGbU9FVUtRa0ZOUTBGeFVYZEVkMWxFVmxJd1ZFRlJTQzlDUVZWM1FYZEZRaTk2UVdSQ1owNVdTRkUwUlVablVWVnBTa2R5YldwdFEyMHJMMUY2SzBSVlVrcEJiUXBVV0hRNWNrZEpkME5uV1VsTGIxcEplbW93UlVGM1NVUlRRVUYzVWxGSloxZExhRGhtVlVaaWEwd3JaVFp4WmtGa1N5dFFVMjVRVTB4NFQxRnRVekowQ2taU1JHVTBlVW81WTJsQlEwbFJRMmgxTlV0RE9HTTFLelJGZDJWeFduZFlhVTVhZUhoalNVSjNkMXB3V1VVMWFFa3lSWGxUV1hBNFJrRTlQUW90TFMwdExVVk9SQ0JEUlZKVVNVWkpRMEZVUlMwdExTMHRDZz09CiAgICBjbGllbnQta2V5LWRhdGE6IExTMHRMUzFDUlVkSlRpQkZReUJRVWtsV1FWUkZJRXRGV1MwdExTMHRDazFJWTBOQlVVVkZTVVZRWldGaVp6ZzNiMUpOZVVkaEsyRldibmRHYmlzMVUxcGxaRU01VEZKUFpITldVbGhLYkV4d1YzQnZRVzlIUTBOeFIxTk5ORGtLUVhkRlNHOVZVVVJSWjBGRk4xcDBSbUowYkVvelJuSXJOQ3RsVmk5SmNITkNVRWMyU25CcVZqYzJNMXA2WXpNNFluTjJlV1UxYzFwbVRtOXlUR0kyTUFwbldscFpSVmM0ZWxkYVJVRXpjRVUzU0VWb1ZUazNURzU2T1hwQk5qRTRjRzVSUFQwS0xTMHRMUzFGVGtRZ1JVTWdVRkpKVmtGVVJTQkxSVmt0TFMwdExRbz0K"
            },
            "kind": "Secret",
            "metadata": {
                "annotations": {
                    "config.k8s.io/owning-inventory": "4a4ffab8fac9214d7b02f1c67e941adf9a37f820-1662338820816906910",
                    "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"data\":{\"kubeconfig\":\"YXBpVmVyc2lvbjogdjEKY2x1c3RlcnM6Ci0gY2x1c3RlcjoKICAgIGNlcnRpZmljYXRlLWF1dGhvcml0eS1kYXRhOiBMUzB0TFMxQ1JVZEpUaUJEUlZKVVNVWkpRMEZVUlMwdExTMHRDazFKU1VKbFJFTkRRVkl5WjBGM1NVSkJaMGxDUVVSQlMwSm5aM0ZvYTJwUFVGRlJSRUZxUVdwTlUwVjNTSGRaUkZaUlVVUkVRbWh5VFROTmRHTXlWbmtLWkcxV2VVeFhUbWhSUkVVeVRsUnJNVTlFU1hwT1ZGRjNTR2hqVGsxcVNYZFBSRUV3VFVSTmQwNVVWVEJYYUdOT1RYcEpkMDlFUVhoTlJFMTNUbFJWTUFwWGFrRnFUVk5GZDBoM1dVUldVVkZFUkVKb2NrMHpUWFJqTWxaNVpHMVdlVXhYVG1oUlJFVXlUbFJyTVU5RVNYcE9WRkYzVjFSQlZFSm5ZM0ZvYTJwUENsQlJTVUpDWjJkeGFHdHFUMUJSVFVKQ2QwNURRVUZTVm5adE4yRkpWM04wYVZVdlJsYzNjMnMyV1VaMk1sRkJVMUpwUW1zNVJGTXJNM2xuVmpaNkwySUtRMkl6WVRKaFlVRXhTbTAxTW5BMlNHOHdOV2xIVmt0TFpIRmxPSHBOZFVNM1VUUlpUVWhxUVRSRVpEZHZNRWwzVVVSQlQwSm5UbFpJVVRoQ1FXWTRSUXBDUVUxRFFYRlJkMFIzV1VSV1VqQlVRVkZJTDBKQlZYZEJkMFZDTDNwQlpFSm5UbFpJVVRSRlJtZFJWVk4xWVhwTU9VOW1SRGxIYmpZMFdVbERhemxRQ2tORWRuSnVRelIzUTJkWlNVdHZXa2w2YWpCRlFYZEpSRk5SUVhkU1owbG9RVXg2VDJ4RVdGVXpVbWx0YUZwTFIxcHBOVkZpUWpremJ6ZHZkbXRCU25RS05rWmpTekZGWlZZeVNqbFNRV2xGUVROd1Jpc3djSFJWY1RrMVlXVlJkbEUzWlV4WVZtWXhObUY1UlVwTk4xTTFSek5IV1ZWTk1tNUhPRFE5Q2kwdExTMHRSVTVFSUVORlVsUkpSa2xEUVZSRkxTMHRMUzBLCiAgICBzZXJ2ZXI6IGh0dHBzOi8vMC4wLjAuMDo2NDQzCiAgbmFtZTogazNkLXJ1bm5lcgpjb250ZXh0czoKLSBjb250ZXh0OgogICAgY2x1c3RlcjogazNkLXJ1bm5lcgogICAgdXNlcjogYWRtaW5AazNkLXJ1bm5lcgogIG5hbWU6IGszZC1ydW5uZXIKY3VycmVudC1jb250ZXh0OiBrM2QtcnVubmVyCmtpbmQ6IENvbmZpZwpwcmVmZXJlbmNlczoge30KdXNlcnM6Ci0gbmFtZTogYWRtaW5AazNkLXJ1bm5lcgogIHVzZXI6CiAgICBjbGllbnQtY2VydGlmaWNhdGUtZGF0YTogTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVSnJha05EUVZSbFowRjNTVUpCWjBsSlJHNVlVVTUwVlhKTmFHdDNRMmRaU1V0dldrbDZhakJGUVhkSmQwbDZSV2hOUWpoSFFURlZSVUYzZDFrS1lYcE9la3hYVG5OaFYxWjFaRU14YWxsVlFYaE9hbFUxVGxSbmVVMTZWVEJOUWpSWVJGUkplVTFFWjNkT1JFRjZUVVJWTVU1R2IxaEVWRWw2VFVSbmR3cE9SRUY2VFVSVk1VNUdiM2ROUkVWWVRVSlZSMEV4VlVWRGFFMVBZek5zZW1SSFZuUlBiVEZvWXpOU2JHTnVUWGhHVkVGVVFtZE9Wa0pCVFZSRVNFNDFDbU16VW14aVZIQm9Xa2N4Y0dKcVFscE5RazFIUW5seFIxTk5ORGxCWjBWSFEwTnhSMU5OTkRsQmQwVklRVEJKUVVKUE1tSlNWemRhVTJSNFlTOTFVRzRLYkdaNVMySkJWSGgxYVdGWk1XVXJkREpqTTA0dlJ6ZE1PRzUxWWtkWWVtRkxlVElyZEVsSFYxZENSblpOTVcxU1FVNDJVazk0ZUVsV1VHVjVOVGd2WXdwM1QzUm1TMW95YWxORVFrZE5RVFJIUVRGVlpFUjNSVUl2ZDFGRlFYZEpSbTlFUVZSQ1owNVdTRk5WUlVSRVFVdENaMmR5UW1kRlJrSlJZMFJCYWtGbUNrSm5UbFpJVTAxRlIwUkJWMmRDVTBscllYVmhUMWxMWWpjNVJGQTBUbEpGYTBOYVRtVXpNbk5aYWtGTFFtZG5jV2hyYWs5UVVWRkVRV2RPU2tGRVFrY0tRV2xGUVdvMmJURkJiazR4ZEVvd2JEUlFUM1JwUkdFM1RXZEhjbEV6ZVVoelFpczNWbTg0Tm05b01XWnlXVzlEU1ZGRGJTOVFhM1ZSVERKcmNHWlllZ3BEUWxScFExWXJUSEEwVFhFd2RHbHZkelZCVW1wcVFrbHNkRWhCT0ZFOVBRb3RMUzB0TFVWT1JDQkRSVkpVU1VaSlEwRlVSUzB0TFMwdENpMHRMUzB0UWtWSFNVNGdRMFZTVkVsR1NVTkJWRVV0TFMwdExRcE5TVWxDWkhwRFEwRlNNbWRCZDBsQ1FXZEpRa0ZFUVV0Q1oyZHhhR3RxVDFCUlVVUkJha0ZxVFZORmQwaDNXVVJXVVZGRVJFSm9jazB6VFhSWk1uaHdDbHBYTlRCTVYwNW9VVVJGTWs1VWF6RlBSRWw2VGxSUmQwaG9ZMDVOYWtsM1QwUkJNRTFFVFhkT1ZGVXdWMmhqVGsxNlNYZFBSRUY0VFVSTmQwNVVWVEFLVjJwQmFrMVRSWGRJZDFsRVZsRlJSRVJDYUhKTk0wMTBXVEo0Y0ZwWE5UQk1WMDVvVVVSRk1rNVVhekZQUkVsNlRsUlJkMWRVUVZSQ1oyTnhhR3RxVHdwUVVVbENRbWRuY1docmFrOVFVVTFDUW5kT1EwRkJWRUZuWWxKUlpFdGhhbkZrUTA5UU5tcEtkV3RvWW14ck5sSkNkVXR0ZWtwbU1rdGhXSE5HVUdsMUNrOVliemQxYzBneGVWbEplbHB4YmpaclNTdHhZVEV2WnpWTVMxVXpja0pQVkRCWGNrNVJWVk41UjFOUGJ6QkpkMUZFUVU5Q1owNVdTRkU0UWtGbU9FVUtRa0ZOUTBGeFVYZEVkMWxFVmxJd1ZFRlJTQzlDUVZWM1FYZEZRaTk2UVdSQ1owNVdTRkUwUlVablVWVnBTa2R5YldwdFEyMHJMMUY2SzBSVlVrcEJiUXBVV0hRNWNrZEpkME5uV1VsTGIxcEplbW93UlVGM1NVUlRRVUYzVWxGSloxZExhRGhtVlVaaWEwd3JaVFp4WmtGa1N5dFFVMjVRVTB4NFQxRnRVekowQ2taU1JHVTBlVW81WTJsQlEwbFJRMmgxTlV0RE9HTTFLelJGZDJWeFduZFlhVTVhZUhoalNVSjNkMXB3V1VVMWFFa3lSWGxUV1hBNFJrRTlQUW90TFMwdExVVk9SQ0JEUlZKVVNVWkpRMEZVUlMwdExTMHRDZz09CiAgICBjbGllbnQta2V5LWRhdGE6IExTMHRMUzFDUlVkSlRpQkZReUJRVWtsV1FWUkZJRXRGV1MwdExTMHRDazFJWTBOQlVVVkZTVVZRWldGaVp6ZzNiMUpOZVVkaEsyRldibmRHYmlzMVUxcGxaRU01VEZKUFpITldVbGhLYkV4d1YzQnZRVzlIUTBOeFIxTk5ORGtLUVhkRlNHOVZVVVJSWjBGRk4xcDBSbUowYkVvelJuSXJOQ3RsVmk5SmNITkNVRWMyU25CcVZqYzJNMXA2WXpNNFluTjJlV1UxYzFwbVRtOXlUR0kyTUFwbldscFpSVmM0ZWxkYVJVRXpjRVUzU0VWb1ZUazNURzU2T1hwQk5qRTRjRzVSUFQwS0xTMHRMUzFGVGtRZ1JVTWdVRkpKVmtGVVJTQkxSVmt0TFMwdExRbz0K\"},\"kind\":\"Secret\",\"metadata\":{\"annotations\":{\"config.k8s.io/owning-inventory\":\"4a4ffab8fac9214d7b02f1c67e941adf9a37f820-1662338820816906910\"},\"name\":\"kubeconfig-target-cluster\",\"namespace\":\"tekton-namespace\"}}\n"
                },
                "creationTimestamp": "2022-09-05T00:47:14Z",
                "labels": {
                    "api.kubernetes.io/name": "secret"
                },
                "name": "kubeconfig-target-cluster",
                "namespace": "tekton-namespace",
                "resourceVersion": "347402",
                "uid": "37320bbf-1ec1-4f7b-8527-9ad482692d95"
            },
            "type": "Opaque"
        }
    ],
    "kind": "List",
    "metadata": {
        "resourceVersion": ""
    }
}
