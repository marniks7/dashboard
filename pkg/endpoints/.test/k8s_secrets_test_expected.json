{
  "kind": "List",
  "apiVersion": "v1",
  "metadata": {},
  "items": [
    {
      "kind": "Secret",
      "apiVersion": "v1",
      "metadata": {
        "name": "default-token-6q424",
        "namespace": "tekton-namespace",
        "resourceVersion": "27862"
      }
    },
    {
      "kind": "Secret",
      "apiVersion": "v1",
      "metadata": {
        "name": "kubeconfig-target-cluster",
        "namespace": "tekton-namespace",
        "labels": {
          "api.kubernetes.io/name": "secret"
        },
        "resourceVersion": "347402"
      }
    }
  ]
}